### PR TITLE
Add WIF to Private key

### PR DESCRIPTION
**Pull request by zolagonano**

### CODE REVIEW

**Review by canewsin — APPROVED** [review]
LGTM

File: src/error.rs [hunk 1]
```diff
@@ -18,6 +18,10 @@ pub enum CryptError {
   AddressMismatch(String),
   #[error("Private key failure")]
   PrivateKeyFailure,
+  #[error("Invalid WIF private key")]
+  InvalidWIFPrivKey,
+  #[error("Invalid network (expected mainnet)")]
+  InvalidNetwork
```
Comment (canewsin) [inline src/error.rs]: Suggested
`InvalidNetwork` -> `InvalidNetworkKey`

**Review by canewsin — CHANGES_REQUESTED** [review]

File: src/error.rs (hunk 1 above)
Comment (zolagonano) [reply src/error.rs]: Done!

Comment (canewsin) [reply src/error.rs]: Update Error macro content also

Comment (zolagonano) [reply src/error.rs]: Done!

**Review by canewsin — APPROVED** [review]

**Pull request merged**